treeherder/perf/management/commands/import_perf_data.py

```diff
@@ -58,6 +58,7 @@ def _add_series(pc, project_name, signature_hash, signature_props, verbosity):
             'platform': platform,
             'framework': framework,
             'extra_properties': extra_properties,
+            'parent_signature': parent_signature,
```

Comment:
You will need to specify the actual parent signature here, not just the hash of it.